lib/runcible/models/yum_importer.rb

```diff
@@ -3,6 +3,13 @@ module Models
     class YumImporter < Importer
       ID = 'yum_importer'
       REPO_TYPE = 'rpm-repo'
+      DOWNLOAD_POLICIES = [
+        DOWNLOAD_IMMEDIATE = 'immediate',
+        DOWNLOAD_ON_DEMAND = 'on_demand',
+        DOWNLOAD_BACKGROUND = 'background'
```

Comment:
I think so. What I have is equivalent to:

```ruby
DOWNLOAD_IMMEDIATE = 'immediate'
DOWNLOAD_ON_DEMAND = 'on_demand'
DOWNLOAD_BACKGROUND = 'background'
DOWNLOAD_POLICIES = [DOWNLOAD_IMMEDIATE, DOWNLOAD_ON_DEMAND, DOWNLOAD_BACKGROUND].freeze
```
I can change it to this if you prefer.